Project/Assets/Prairie/Framework/Script/Interaction/FirstPersonInteractor.cs

```diff
@@ -39,6 +39,16 @@ void Update ()
 			// right click
 			this.AttemptReadAnnotation ();
 		}
+		if (areaAnnotationsInRange.Count != 0)
+		{
+			for (int a = 0; a < areaAnnotationsInRange.Count; a++)
+			{
+				if (Input.GetKeyDown ((a+1).ToString()))
+				{
```

Comment:
Clever way to avoid typing a bunch of manual checks (such as `Input.GetKeyDown(KeyCode.one)`, `Input.GetKeyDown(KeyCode.two)`, ...)